src/components/BottomSheetInLineNotification.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import { StyleSheet } from 'react-native'
+import Animated, { useAnimatedStyle, useSharedValue } from 'react-native-reanimated'
+import InLineNotification, { InLineNotificationProps } from 'src/components/InLineNotification'
+import Modal from 'src/components/Modal'
+import { useShowOrHideAnimation } from 'src/components/useShowOrHideAnimation'
+import { Spacing } from 'src/styles/styles'
+
+interface Props extends InLineNotificationProps {
+  showNotification: boolean
+}
+
+// this value is used to ensure the notification is offset by its own height when transitioning in and out of view
+const NOTIFICATION_HEIGHT = 200
```

Comment:
This seems could cause issues if the notification height is bigger than this hardcoded value.
For instance what happens if the text is longer in German?

Reply:
i increased this value to account for a buffer and made the comment more detailed. if this value is greater than the toast height, it doesn't really matter (the animation happens a bit faster, it's not so noticable). if the value is less than the toast height, the animation doesn't finish in time but it's still not a terrible experience...